docs/framework/whats-new/index.md

```diff
@@ -30,23 +31,107 @@ This article does not provide comprehensive information about each new feature a
 > [!NOTE]
 > The .NET Framework team also releases features out of band with NuGet to expand platform support and to introduce new functionality, such as immutable collections and SIMD-enabled vector types. For more information, see [Additional Class Libraries and APIs](../additional-apis/index.md) and [The .NET Framework and Out-of-Band Releases](~/docs/framework/get-started/the-net-framework-and-out-of-band-releases.md). See a [complete list of NuGet packages](https://blogs.msdn.microsoft.com/dotnet/p/nugetpackages/) for the .NET Framework, or subscribe to [our feed](https://nuget.org/api/v2/curated-feeds/dotnetframework/Packages/).
 
-<a name="v47"></a> 
-## Introducing the .NET Framework 4.7
+<a name="v471"></a> 
+## Introducing the .NET Framework 4.7.1
 
-The .NET Framework 4.7 builds on the .NET Framework 4.6, 4.6.1, and 4.6.2 by adding many new fixes and several new features while remaining a very stable product.
+The .NET Framework 4.7.1 builds on the .NET Framework 4.6, 4.6.1, 4.6.2, and 4.7 by adding many new fixes and several new features while remaining a very stable product.
 
 ### Downloading and installing the .NET Framework 4.7
```

Comment:
4.7 -> 4.7.1?

Reply:
No, this is correct: 4.7.1 builds on 4.7

Reply:
Sure, but then you immediately follow with download 4.7.1. It seems like a typo.

![image](https://user-images.githubusercontent.com/12971179/31627672-e6746492-b262-11e7-945d-441bf346d9ef.png)

Reply:
Ah, I missed that.